LibreHardwareMonitorLib/Hardware/Memory/AmdDimmSensor.cs

```diff
@@ -0,0 +1,134 @@
+﻿// This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0.
+// If a copy of the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+// Copyright (C) LibreHardwareMonitor and Contributors.
+// Partial Copyright (C) Michael Möller <mmoeller@openhardwaremonitor.org> and Contributors.
+// All Rights Reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+
+namespace LibreHardwareMonitor.Hardware.Memory
+{
+    internal sealed class AmdDimmSensor : DimmSensor
+    {
+        public AmdDimmSensor(string name, int index, Hardware hardware, ISettings settings, byte address) : base(name, index, hardware, settings, address)
+        {
+
+        }
+
+        public override void UpdateSensor()
+        {
+            try
+            {
+                ushort data = GetWord(_address, 0x05);
```

Comment:
What's the point in composing BE word in `GetPort()` if you always switch bytes afterwards?